osu.Game/Storyboards/Drawables/DrawableStoryboardAnimation.cs

```diff
@@ -118,8 +118,24 @@ private void load(TextureStore textureStore, Storyboard storyboard)
             for (int frame = 0; frame < Animation.FrameCount; frame++)
             {
                 string framePath = Animation.Path.Replace(".", frame + ".");
+                Drawable sprite = storyboard.CreateSpriteFromResourcePath(framePath, textureStore);
 
-                AddFrame(storyboard.CreateSpriteFromResourcePath(framePath, textureStore), Animation.FrameDelay);
+                if (sprite != null)
+                {
+                    AddFrame(sprite, Animation.FrameDelay);
+                    continue;
+                }
+
+                framePath = Animation.Path.Replace("0.", frame + ".");
+                sprite = storyboard.CreateSpriteFromResourcePath(framePath, textureStore);
```

Comment:
And the "first sprite with no appended 0" crash is resolved by checking for that scenario if the previous check did not find a sprite

Reply:
what's the point of the second replace? can't it just be `Animation.Path`?